app/Services/Credit.php

```diff
@@ -90,6 +90,15 @@ public function getBalance()
         return $this->user->cash_balance / 100;
     }
 
+    /**
+     * Get the users balance sign (positive/negative)
+     * @return string
+     */
+    public function getBalanceSign()
+    {
+		return ( (int) $this->getBalance() >= 0 ? 'positive' : 'negative' );
```

Comment:
All looks brilliant except for the tab indentation!
Can you switch these to spaces (4 space tabs) please?